packages/wonder-blocks-data/src/hooks/__tests__/use-data.test.js

```
@@ -446,7 +448,99 @@ describe("#useDataInternal", () => {
 
             // Act
             const render = clientRenderHook(
-                ({options}) => useDataInternal(oldHandler, options),
+                ({options}) => useData(oldHandler, options),
+                {
+                    initialProps: {options: "OLD OPTIONS"},
+                },
+            );
+            render.rerender({options: "NEW OPTIONS"});
+            await act(() => oldRequest.catch(() => {}));
```

The `.catch()` is here just to ignore errors, right?